plugins/atomicfu/atomicfu-compiler/testData/box/ComplexLoopTest.txt

```diff
@@ -0,0 +1,44 @@
+@kotlin.Metadata
+public final class ComplexLoopTestKt {
+    // source: 'ComplexLoopTest.kt'
+    public final static @org.jetbrains.annotations.NotNull method box(): java.lang.String
+}
+
+@kotlin.Metadata
+public final class LoopTest {
+    // source: 'ComplexLoopTest.kt'
+    private final static @org.jetbrains.annotations.NotNull field a$FU: java.util.concurrent.atomic.AtomicIntegerFieldUpdater
+    private volatile volatile @kotlin.jvm.Volatile field a: int
```

Comment:
@udalov, Looks like we have a bug in our bytecode listing dumper - it duplicates 'volatile‘.

Reply:
Indeed. I guess it's because of this line which is not needed anymore: https://github.com/JetBrains/kotlin/blob/00d7a755a9fb311e66e28217f7c1e768c9b2dac7/compiler/tests-compiler-utils/tests/org/jetbrains/kotlin/codegen/BytecodeListingTextCollectingVisitor.kt#L242. @mvicsokolova Please consider deleting it.

Weird that this is the first place where we observe this. There are no words "volatile" in all txt dumps of `compiler/testData/codegen`.

Reply:
Thank you! Removed this line, all volatiles are printed once now.